lib/utils.js

```diff
@@ -116,7 +116,7 @@ module.exports = {
     getVersions() {
         const packageJson = require("../package.json");
         const packageVersion = packageJson.version;
-        const context = this.getContext();
+        const context = this.getEnvironment();
```

Comment:
If we rename this to environment, we should be consistent in the naming throughout the code.
```suggestion
        const environment = this.getEnvironment();
```